app/controllers/foo_controller.rb

```diff
@@ -0,0 +1,12 @@
+class FooController < ApplicationController
+  def index
+  end
+
+  def is_string?(string)
```

Comment:
__[reek]__
* [UtilityFunction] doesn't depend on instance state (maybe move it to another class?) [:ref](https://github.com/troessner/reek/blob/master/docs/Utility-Function.md)

<!-- Comment by [SideCI](https://sideci.com) -->